services/reference/linea/json-rpc-methods/eth_call.mdx

```diff
@@ -1,40 +1,14 @@
 ---
 title: "eth_call"
+hide_title: true
+hide_table_of_contents: true
 ---
 
-import Tabs from '@theme/Tabs';
-import TabItem from '@theme/TabItem';
+import ParserOpenRPC from "@site/src/components/ParserOpenRPC"
```

Comment:
The enum list is intermingled with the description 
<img width="1075" alt="image" src="https://github.com/user-attachments/assets/03729b8b-ccee-4d1b-a3d6-2693b9f042a6">
Pulled, did install, and built 5 mins before this comment

Reply:
fixed